### PR TITLE
 Translate Ruby 2.5.0 release post (zh_tw)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_tw/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+lang: zh_tw
+---
+
+我們高興地宣佈 Ruby 2.5.0 已發佈。
+
+Ruby 2.5.0 是 Ruby 2.5.0 的第一個穩定版本。此版本包括數個新功能和效能提升。例如：
+
+## 新功能
+
+- 允許在 do/end 區塊中 rescue/else/ensure。[[功能 #12906]](https://bugs.ruby-lang.org/issues/12906)
```
Comment (stomar) [inline zh_tw/news/_posts/2017-12-25-ruby-2-5-0-released.md]: markdown style (* for bullet points)

File: zh_tw/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- ERB 現在模板生成原始碼的速度相較於 Ruby 2.4 提升了 2 倍。
+- 提升了多個內建方法，例如 `Array#concat`, `Enumerable#sort_by`, `String#concat`, `String#index`, `Time#+` 等的效能。
+- IO.copy\_stream 使用 copy\_file\_range(2) 來拷貝負載。[[功能 #13867]](https://bugs.ruby-lang.org/issues/13867)
+
+## 自 2.4 起重要的變化
+
+- SecureRandom 傾向使用作業系統方法而非 OpenSSL。[[漏洞 #9569]]((https://bugs.ruby-lang.org/issues/9569))
+- 將 cmath, csv, date, dbm, etc, fcntl, fiddle, fileutils, gdbm, ipaddr, scanf, sdbm, stringio, strscan, webrick, zlib 從函式庫提升到預設 gem。
+- 升級 [Onigmo](https://github.com/k-takata/Onigmo/) 到 6.1.3。
+  - 新增 [absent operator](https://github.com/k-takata/Onigmo/issues/87)。
```
Comment (stomar) [inline zh_tw/news/_posts/2017-12-25-ruby-2-5-0-released.md]: absence operator

**Comment by dsh0416** [comment]
bullet points fixed. Also fixed the incorrect **abscent operator** in 2.5.0.rc1 translation

**Comment by stomar** [comment]
Closing because of merge commit.

Please provide a feature branch with a single commit; and please use the common commit message style (like this PR title).

**Pull request closed without merging**